roles/openshift_node/templates/node.yaml.v1.j2

```diff
@@ -18,9 +18,17 @@ kubeletArguments: {{  l2_openshift_node_kubelet_args  | default(None) | lib_util
   container-runtime:
   - remote
   container-runtime-endpoint:
+{% if openshift.common.version_gte_3_9 %}
   - /var/run/crio/crio.sock
```

Comment:
We can also substitute /var/run/crio/crio.sock with a variable and avoid these if statements in the various templates.  We should resolve this logic in role/defaults.yml.